state/runtime/runtime.go

```diff
@@ -130,20 +131,40 @@ func (r *ExecutionResult) UpdateGasUsed(gasLimit uint64, refund uint64) {
 
 var (
 	ErrOutOfGas                 = errors.New("out of gas")
-	ErrStackOverflow            = errors.New("stack overflow")
-	ErrStackUnderflow           = errors.New("stack underflow")
 	ErrNotEnoughFunds           = errors.New("not enough funds")
 	ErrInsufficientBalance      = errors.New("insufficient balance for transfer")
-	ErrMaxCodeSizeExceeded      = errors.New("evm: max code size exceeded")
+	ErrMaxCodeSizeExceeded      = errors.New("max code size exceeded")
 	ErrContractAddressCollision = errors.New("contract address collision")
 	ErrDepth                    = errors.New("max call depth exceeded")
-	ErrExecutionReverted        = errors.New("execution was reverted")
+	ErrExecutionReverted        = errors.New("execution reverted")
 	ErrCodeStoreOutOfGas        = errors.New("contract creation code storage out of gas")
 	ErrUnauthorizedCaller       = errors.New("unauthorized caller")
 	ErrInvalidInputData         = errors.New("invalid input data")
 	ErrNotAuth                  = errors.New("not in allow list")
 )
```

Comment:
Are we lacking these error messages and should we introduce them:

- https://github.com/ethereum/go-ethereum/blob/master/core/vm/errors.go#L32
`ErrMaxInitCodeSizeExceeded  = errors.New("max initcode size exceeded")`

- https://github.com/ethereum/go-ethereum/blob/master/core/vm/errors.go#L38
`ErrInvalidCode              = errors.New("invalid code: must not begin with 0xef")`

- https://github.com/ethereum/go-ethereum/blob/master/core/vm/errors.go#L39
`ErrNonceUintOverflow        = errors.New("nonce uint64 overflow")`

Of course we should merge this one, but just thinking outloud whether we should introduce those as well.